actions/dataminrpulse_get_alerts.py

```diff
@@ -0,0 +1,92 @@
+# File: dataminrpulse_get_alerts.py
+#
+# Copyright (c) 2023 Dataminr
+#
+# This unpublished material is proprietary to Dataminr.
+# All rights reserved. The methods and
+# techniques described herein are considered trade secrets
+# and/or confidential. Reproduction or distribution, in whole
+# or in part, is forbidden except by express written permission
+# of Dataminr.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+
+import phantom.app as phantom
+
+import dataminrpulse_consts as consts
+from actions import BaseAction
+
+
+class GetAlertsAction(BaseAction):
+    """Class to handle get alerts action."""
+
+    def execute(self):
+        """Execute the  get alerts action."""
+        list_id = self._param.get("list_id", None)
+        use_asset_configured_lists = self._param.get("use_asset_configured_lists", False)
+        query = self._param.get("query", None)
+
+        # Accepts comma-seperated list only
+        if list_id:
+            list_id = list_id.strip(',')
+            list_id = str(list_id).replace(" ", '')
```

Comment:
```suggestion
            list_id = str(list_id).replace(' ', '')
```
Use quote consistency please

Reply:
Done